### DISCUSSION TITLE
Tech stack of Magicbell API

### DISCUSSION THREAD

**Discussion by abhisheksunil2201** [magicbell/community#62] [General]
I was curious about how MagicBell handles the amount of requests coming to the API and the tech stack that has been used to build the API. I used it for one of my project and the responses are so reliable.

**Comment by unamashana** [comment]
Thank you very much for the lovely question @abhisheksunil2201  :)

The stack at the moment is Ruby on Rails deployed on AWS using their container service (ECS). We also use a load balancer to front the requests and auto-scale the number of containers based on traffic spikes. Going forward, we plan to move some endpoints over to lambda that can scale up even faster as the traffic spikes up or down. 

Please follow us on Twitter, where we post technical threads every now and then - https://twitter.com/magicbell_io
